bridge/client/app/_views/ktb-service-view/ktb-service-view.component.ts

```diff
@@ -64,6 +66,32 @@ export class KtbServiceViewComponent implements OnInit, OnDestroy {
       this._changeDetectorRef.markForCheck();
     });
 
+    combineLatest([params$, project$]).pipe(take(1)).subscribe(([params, project]) => {
+      if (params.shkeptncontext && params.serviceName) {
+        const service = project.getServices().find(s => s.serviceName === params.serviceName);
+        const paramDeployment = service.deployments.find(deployment => deployment.shkeptncontext === params.shkeptncontext);
```

Comment:
Just a nitpick ... as filter returns an array consider it renaming to changedDeployments